checks-superstaq/checks_superstaq/pytest_.py

```diff
@@ -71,13 +70,10 @@ def run(
     files = check_utils.extract_files(parsed_args, include, exclude, silent)
 
     if parsed_args.notebook:
-        args_to_pass += ["--nbmake"]
+        args_to_pass += ["--nbmake", "--force-enable-socket"]
     elif not parsed_args.integration:
         files = check_utils.get_test_files(files, exclude=exclude, silent=silent)
```

Comment:
nit: can we keep the two `force-enable-socket` additions together this block, e.g.
```suggestion
    elif parsed_args.integration:
        args_to_pass += ["--force-enable-socket"]
    else:
        files = check_utils.get_test_files(files, exclude=exclude, silent=silent)
```
instead of adding the integration version in the block above?

Reply:
Updated to include change